_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1066,6 +1075,95 @@ private function _process_onboarding( $data ) {
 			: join( ', ', $error );
 	}
 
+	/**
+	 * Retrieve the first active sidebar.
+	 *
+	 * @return array|WP_Error First active sidebar, error if none exists.
+	*/
+	static function get_first_sidebar() {
+		$active_sidebars = get_option( 'sidebars_widgets', array() );
+		unset( $active_sidebars[ 'wp_inactive_widgets' ], $active_sidebars[ 'array_version' ] );
+
+		if ( empty( $active_sidebars ) ) {
+			return new WP_Error( 'invalid_data', 'Invalid data.', 400 );
```

Comment:
I think this is not a case when we want to return a `WP_Error`; this is a case when we want to fail silently because there are no sidebars registered. Returning `false` should probably make more sense, especially because down there have `if ( $first_sidebar ) {` when inserting/updating the widget.

Reply:
Updated to `false`. 
Side note: from a user perspective it does not feel like a good UX to not inform them why the business address will not appear on their site despite having completed the step. Should we possibly enable a sidebar on their behalf?